examples/ios/Examples/RealmApp.swift

```diff
@@ -68,6 +71,58 @@ class RealmAppTest: XCTestCase {
         }
     }
     
+    func testRealmAppWithCustomBaseURL() async {
+        // Note for internal review: I tested this manually running an Edge Server
+        // on my machine and it succeeds.
+        // Until we get Edge Server running in a CI to test against, this will
+        // fail in CI. So I'm going to expect this test to fail.
+        
+        // :snippet-start: custom-base-url
+        // Specify a baseURL to connect to a server other than the default.
+        // In this case, an Edge Server instance running on the device.
+        let configuration = AppConfiguration(baseURL: "http://localhost:80")
+
+        let edgeApp = App(id: EDGE_SERVER_APP_ID, configuration: configuration)
+        // :snippet-end:
+        do {
+            let user = try await edgeApp.login(credentials: Credentials.anonymous)
+            XCTAssert(user.isLoggedIn)
+        } catch {
+            let error = error.localizedDescription
+            XCTAssertEqual(error, "Could not connect to the server.")
+        }
+    }
+    
+    func testChangeBaseURL() async {
+        // Note for internal review: I tested this manually running an Edge Server
+        // on my machine and it succeeds. Until we get Edge Server running in a CI
+        // I can't login to an Edge Server in an automated test, so I can't write
+        // automated tests for the full flow.
+        
+        // Specify a baseURL to connect to an Edge Server instance running on the device
+        let configuration = AppConfiguration(baseURL: "http://localhost:80")
+        
+        do {
+            // :snippet-start: change-base-url
+            // Specify a baseURL to connect to a server other than the default.
+            // In this case, an Edge Server instance running on the device.
+            let configuration = AppConfiguration(baseURL: "http://localhost:80")
+            let edgeApp = App(id: EDGE_SERVER_APP_ID, configuration: configuration)
+            
+            // ... log in a user and use the app...
+            // ... some time later...
+            
```

Comment:
```suggestion
          // ... log in a user and use the app...
          
          // Update the baseURL to, in this example, use the global default.
```
Suggestion: Delete 2nd comment that is not as necessary and add comment above baseURL update